src/mon/Monitor.h

```diff
@@ -170,6 +173,9 @@ class Monitor : public Dispatcher,
 private:
   void new_tick();
 
+  // Memory usage baseline snapshot for monitoring
+  MemoryModel::snap baseline;
```

Comment:
Please also fix MDCache to not have a `static` duration variable: https://github.com/ceph/ceph/pull/56812#discussion_r1566250407